### PR TITLE
Cleanup/main.py

### PR DESCRIPTION
**Pull request by ReasonablySelenium**
Cleans up main.py and adds embeds/help.json to handle the help function.

### CODE REVIEW

**Review by OfekShochat — COMMENTED** [review]
what is that embeds/about.json file? seems to be empty.

**Review by OfekShochat — CHANGES_REQUESTED** [review]
please change that embeds/about.json file to be fitted to the `about` function in main.py. when that's met, this or will be merged. 
thank you for contributing,
Ofek

**Comment by OfekShochat** [comment]
Thank you so much. Will look at it when at home!

**Review by OfekShochat — CHANGES_REQUESTED** [review]

File: main.py [hunk 1]
```diff
@@ -83,82 +96,100 @@ async def calculate(ctx, *args):
...
+        if cont:
             cont = False
             continue
 
         try:
             float(i)
             command += i
-        except:
-            print("syntax error")
+        except SyntaxError as e:
```
[the remark this answers is not in the document]
Comment (OfekShochat) [reply main.py:106]: also if you dont mind, send that to the discord chat instead of printing

[the remark this answers is not in the document]
Comment (ReasonablySelenium) [reply main.py]: I made a change to the code. Please review.

**Review by OfekShochat — APPROVED** [review]

**Comment by OfekShochat** [comment]
thank you for contributing to minimal

**Pull request merged**